{"name": "test_alert3_failure", "status": "broken", "statusDetails": {"message": "NameError: name 'WaitUtils' is not defined", "trace": "self = <test_alerts.TestAlertsPage object at 0x000001D6E0AFEE00>\n\n    @allure.feature(\"Alerts\")\n    @allure.story(\"Тест второго Alert (Негативный - нажатие Отмены)\")\n    def test_alert3_failure(self):\n>       self.alert_page.alert3_click()\n\ntest_alerts.py:74: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.alerts_page.AlertsPage object at 0x000001D6E0BE0980>\n\n    def alert3_click(self):\n>       a = WaitUtils.wait(self.browser, (ALERT3[0], ALERT3[1]))\n            ^^^^^^^^^\nE       NameError: name 'WaitUtils' is not defined\n\npages\\alerts_page.py:52: NameError"}, "attachments": [{"name": "stdout", "source": "223e8295-b0f5-4981-b360-cfe5ea1c155a-attachment.txt", "type": "text/plain"}], "start": 1761746048228, "stop": 1761746048229, "uuid": "482b896c-7ab9-4ad8-be43-666c5cdd4af0", "historyId": "168378ac3bb54ecc85339052d470cf0c", "testCaseId": "168378ac3bb54ecc85339052d470cf0c", "fullName": "test_alerts.TestAlertsPage#test_alert3_failure", "labels": [{"name": "story", "value": "Тест второго Alert (Негативный - нажатие Отмены)"}, {"name": "feature", "value": "Alerts"}, {"name": "suite", "value": "test_alerts"}, {"name": "subSuite", "value": "TestAlertsPage"}, {"name": "host", "value": "DESKTOP-T28TT8C"}, {"name": "thread", "value": "18156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alerts"}], "titlePath": ["test_alerts.py", "TestAlertsPage"]}